.vscode/settings.json

```diff
@@ -6,6 +6,19 @@
         "**/dist": true
     },
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
-    "eslint.validate": ["typescript", "typescriptreact", "javascript"]
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "eslint.validate": [
+        "typescript",
+        "typescriptreact",
+        "javascript"
+    ],
+    "[typescript]": {
+        "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+    },
+    "[javascript]": {
+        "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+      },
+      "[typescriptreact]": {
+        "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+      },
```

Comment:
Linting doesn't work correctly on my machine, but that will be taken care of by [#289](https://github.com/equinor/mad/issues/289).